stacktrace_test.go

```diff
@@ -27,14 +27,7 @@ import (
 )
 
 func TestTakeStacktrace(t *testing.T) {
-	// Even if we pass a tiny buffer, takeStacktrace should allocate until it
-	// can capture the whole stacktrace.
-	traceNil := takeStacktrace(nil, false)
-	traceTiny := takeStacktrace(make([]byte, 1), false)
-	for _, trace := range []string{traceNil, traceTiny} {
-		// The top frame should be takeStacktrace.
-		assert.Contains(t, trace, "zap.takeStacktrace", "Stacktrace should contain the takeStacktrace function.")
-		// The stacktrace should also capture its immediate caller.
-		assert.Contains(t, trace, "TestTakeStacktrace", "Stacktrace should contain the test function.")
-	}
+	trace := takeStacktrace()
+	// The stacktrace should also its immediate caller.
+	assert.Contains(t, trace, "TestTakeStacktrace", "Stacktrace should contain the test function.")
```

Comment:
since the test is in the `zap` package, why does it appear in the output of `takeStacktrace`?